.gitlab/conf/host-configs/blueos_3_ppc64le_ib_p9

```diff
@@ -0,0 +1 @@
+blueos_3_ppc64le_ib
```

Comment:
Is this file complete?

Reply:
Yes. 

This file is a symbolic link to `blueos_3_ppc64le_ib` directory so that the configuration for `blueos_3_ppc64le_ib_p9` is the same as `blueos_3_ppc64le_ib`.

The mechanism is the same as it the project host-configs directory.